### ISSUE TITLE
Allow default args in arguments object

### ISSUE THREAD

**Issue by kolodny** [babel/babel#2538]
The following code fails:

```js
const foo = async ({bar = 'bar' }) => {
}
```

While this works as expected:

```js
const foo = ({bar = 'bar' }) => {
}
```

This is the error message:

```
Uncaught SyntaxError: repl: Unexpected token (1:24)
> 1 | const foo = async ({bar = 'bar' }) => {
    |                         ^
  2 | }
```

[link to repl](http://babeljs.io/repl/#?experimental=false&evaluate=true&loose=false&spec=false&code=const%20foo%20%3D%20async%20(%7Bbar%20%3D%20'bar'%20%7D)%20%3D%3E%20%7B%0A%7D%0A)

**Comment by kolodny** [comment]
As @loganfsmyth pointed out on slack this works

```js
const foo = async ({bar: bar = 'bar' }) => {
}
```

**Comment by danez** [comment]
This was fixed in 6.9.1 babel/babylon#96
